assembly/__tests__/Int.spec.ts

```diff
@@ -53,10 +54,15 @@ describe("Int8", () => {
         expect<Int8>(Int8.fromU8a([0x01])).toStrictEqual(new Int8(1));
         expect<Int8>(Int8.fromU8a([0xf])).toStrictEqual(new Int8(15));
         expect<Int8>(Int8.fromU8a([0x10])).toStrictEqual(new Int8(16));
-        expect<Int8>(Int8.fromU8a([0x7f])).toStrictEqual(new Int8(127));
-        expect<Int8>(Int8.fromU8a([0xff])).toStrictEqual(new Int8(-1));
-        expect<Int8>(Int8.fromU8a([0xf1])).toStrictEqual(new Int8(-15));
-        expect<Int8>(Int8.fromU8a([0x81])).toStrictEqual(new Int8(-127));
+        const int8 = new Int8();
```

Comment:
I think that it would be better to have separate tests for the `populateFromBytes` functionality. Currently, you are testing part of the cases using the `fromU8a` and the other part with `populateFromBytes`. 

Ideally, we want to test both `fromU8a` and `populateFromBytes` with a full set of cases. Meaning that we should have fromU8a test for 1, 15, 16, 127, -1, -15 and -127 and the same tests for `populateFromBytes`.

The same thing is applicable to the other types i16, i32, i64, string, u8, u16, u32, u64